### ISSUE TITLE
dependentSchemas and dependentRequired keywords not implemented

### ISSUE THREAD

**Issue by gjvoosten** [networknt/json-schema-validator#321]
From JSON Schema 2019-09, the new keywords `dependentSchemas` and `dependentRequired` (replacing `dependencies`) are not implemented; see https://json-schema.org/draft/2019-09/release-notes.html#semi-incompatible-changes

**Comment by stevehu** [comment]
@gjvoosten Thanks a lot for raising it up. Several keywords are not implemented as our team is not using them at the moment. They are in the pipeline though when we have time. If would be great if you and others can take the time to get them implemented.

**Comment by kmalski** [comment]
Hi, I have created pull request resolving this issue #479.

**Comment by stevehu** [comment]
@gjvoosten, the implementation provided by @kmalski has merged to the master branch already. Once several other pending issues are resolved, we will have a new release. Meanwhile, you can test the new keyword with the master branch code. Thanks @kmalski for your help.

**Comment by gjvoosten** [comment]
Hi @stevehu, apologies for the delay in getting back to you. Release 1.0.64 works regarding the way we use `dependentSchemas`. [But note that we'd also like to use `unevaluatedProperties` which is part of issue #276 .]

**Comment by stevehu** [comment]
@gjvoosten Yes. You are right. That is another keyword we need to implement. Let's see if someone can help. Thanks for the reminder.

**Comment by stevehu** [comment]
@rustermi Sorry, I don't think this keyword is implemented yet. I would be very happy if someone can submit a PR to get it done. Thanks.

**Comment by rustermi** [comment]
We currently don't use the keyword either. When we do, I will follow your proposal.
